tools/crm_mon.c

```diff
@@ -705,7 +705,7 @@ mon_cib_connection_destroy(gpointer user_data)
         reconnect_timer = g_timeout_add(options.reconnect_ms,
                                         reconnect_after_timeout, NULL);
     }
-    return;
+    refresh_after_event(FALSE, TRUE);
```

Comment:
might as well add this situation to the refresh_after_event() doxygen

Reply:
That lists the conditions when the screen is actually redrawn, assuming that `refresh_after_event()` has already been called. It doesn't list the call conditions

Reply:
Wouldn't hurt to add the "`enforce` is true" condition regardless

Reply:
ah